src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,36 @@
+.stopwatch {
+
+  &__minutes {
+    @include arrow(200px, #c25d5d);
+
+    animation: arrow-movement 3600s linear infinite;
```

Comment:
Create a variable for 3600s like $hour based on $minute. It will be like $hour: $minute * 60 (or even math formula for ^2)